demos/node-classification/node2vec/requirements.txt

```diff
@@ -0,0 +1,2 @@
+paramiko==2.4.2
```

Comment:
The demo runs fine in an environment without `paramiko` installed. I don't understand why `paramiko` is needed, especially since it's a python package for ssh2.